### ISSUE TITLE
"Cannot publish on inactive topic" / "no access wo messages" during chat with a pending websocket request

### ISSUE THREAD

**Issue by pwFoo** [tinode/chat#165]
### Subject of the issue
During chat with an offline user (1:1 chat). I get the error message "Cannot publish on inactive topic" and the message stays in "sending" state. 

### Is this a bug report of a feature request?
- [ x] Bug report
- [ ] Feature request

#### Server-side
- [ ] api.tinode.co
- [ x] Your own setup:
  * linux, docker
  * `0.15.7-rc4`
  * mysql container
  
#### Client-side
- [x ] TinodeWeb/tinodejs: javascript client
  * Chrome 69.0.3497.100 (Offizieller Build) (64-Bit)
- [ ] Tindroid: Android app
  * Android API level (e.g. 25).
  * Emulator or hardware, if hardware describe it.
- [ ] tn-cli
  * Python version
- [ ] Chatbot
  * Python version
- Version of the client: `0.15.7-rc4`

### Steps to reproduce
During send multiple messages to a (offline) user suddenly error message appears and the message and all following messages stuck in "sending". Reload tinode client fix the problem.

### Expected behaviour
Message should be send or send with a little delay (if it is a connection problem).

### Actual behaviour
Stucks in "sending", refresh browser and the message is lost

### Server-side log
I don't know the log part of the pending websocket request (maybe not reached the server and isn't logged?), but here is the request sent after / during pending websocket request.
```
2018/10/05 07:21:32 in: '{"sub":{"id":"105146","topic":"usrUUuWxAjzSJc","get":{"data":{"since":109,"limit":24},"sub":{"ims":"2018-10-05T07:16:13.242Z"},"desc":{"ims":"2018-10-05T07:16:13.242Z"},"del":{"since":1},"what":"data sub desc del"}}}' ip='10.42.127.111:53960' sid='LV2uHc6n_xA' uid=''
2018/10/05 07:21:49 in: '{"sub":{"id":"105147","topic":"usrUUuWxAjzSJc","get":{"data":{"since":109,"limit":24},"sub":{"ims":"2018-10-05T07:16:13.242Z"},"desc":{"ims":"2018-10-05T07:16:13.242Z"},"del":{"since":1},"what":"data sub desc del"}}}' ip='10.42.127.111:53960' sid='LV2uHc6n_xA' uid=''
```

### Client-side log
Looks like a pending websocket request stucks forever?
```
Request URL: wss://<DOMAIN>/v0/channels?apikey=<KEY>
Request Method: GET
Status Code: 101 Switching Protocols
```

If I leave the chat and open it again I get an auth error 401 ("no access to messages").
Websocket request ist still pending. Maybe it's related to my poor internet connection here, but application should handle that (timeout, retry)?

**Comment by or-else** [comment]
Does it happen when the connection to server is lost? 
What do you see in the browser console (Options -> More Tools -> Developer tools -> Console) when it happens? 

Regarding this:
```
Request URL: wss://<DOMAIN>/v0/channels?apikey=<KEY>
Request Method: GET
Status Code: 101 Switching Protocols
```
This is completely normal. Websocket is not HTTP. What you see here is the first step of converting an HTTP connection to a websocket. All subsequent communication is not visible in the browser Network panel. You have the latest Chrome. You can see the websocket communication if you choose `Frames`:

<img width="734" alt="screenshot 2018-10-05 at 12 34 42" src="https://user-images.githubusercontent.com/5938143/46527948-30dbd700-c89b-11e8-812a-c90a0f9c59bb.png">

You can also see the client-side logs in the Console.
<img width="614" alt="screenshot 2018-10-05 at 12 37 13" src="https://user-images.githubusercontent.com/5938143/46528036-6e406480-c89b-11e8-895b-39f16db3ca89.png">

**Comment by or-else** [comment]
A partial fix was released in 0.15.7-rc5

**Comment by or-else** [comment]
It's been fixed for a while. Closing this issue. If something is not working, please open a new issue.
